src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -194,6 +199,18 @@ private void AddSubmissionProjectNoLock(ITextBuffer submissionBuffer, string lan
             _currentSubmissionProjectId = ProjectId.CreateNewId(newSubmissionProjectName);
             var newSubmissionDocumentId = DocumentId.CreateNewId(_currentSubmissionProjectId, newSubmissionProjectName);
 
+            // Associate a name with both the editor document and our roslyn document so LSP can make requests on it.
+            var extension = languageName switch
+            {
+                LanguageNames.CSharp => "cs",
+                LanguageNames.VisualBasic => "vb",
+                _ => throw new ArgumentException($"Unexpected language name {languageName}")
+            };
+            var submissionFilePath = Path.Combine(Path.GetTempPath(), $"Submission{SubmissionCount}.{extension}");
```

Comment:
>Path.GetTempPath() [](http://example.com/codeflow?start=50&length=18)

This is a bit concerning. Wouldn't the editor pick up .editorconfig settings from the temp path, if someone had an .editorconfig file there by accident?

Reply:
That is possible, but wouldn't that apply to the path today as well (e.g C:\Temp.txt)?  Though perhaps an editorconfig file in the users folder is more common.   Any suggestions for a different path?